### PR TITLE
📖 book: drop outdated note about removed e2e test func

### PR DESCRIPTION
**Pull request by sbueringer**
Signed-off-by: Stefan Büringer buringerst@vmware.com

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

### CODE REVIEW

File: docs/book/src/developer/e2e.md [hunk 1]
```diff
@@ -77,16 +77,6 @@ This method:
 - Waits for the providers controllers to be running.
 - Creates log watchers for all the providers
 
-<aside class="note">
```
Comment (sbueringer) [inline docs/book/src/developer/e2e.md:80]: I don't know when it has been removed, but it doesn't exist anymore

**Comment by fabriziopandini** [comment]
nice catch!
/lgtm
/approve

**Comment by fabriziopandini** [comment]
not sure if we should cherry pick also in 1.2
@sbueringer ?

**Comment by sbueringer** [comment]
> not sure if we should cherry pick also in 1.2 @sbueringer ?

I would personally only backport the book into 1.2 for must-have documentation updates. But absolutely no objections if you want to.

**Comment by fabriziopandini** [comment]
> I would personally only backport the book into 1.2 for must-have documentation updates (I'm assuming that most users only read our "current' book). But absolutely no objections if you want to.

I kind of agree, but unfortunately, we don't have currently a good definition of "must-have documentation updates".
might be it will be enough to specify in our backport policies that we backport document changes only to the current version of the book (backport to older releases can be exceptionally approved by maintainers under case-by-case evaluation), or something similar

**Comment by sbueringer** [comment]
Good point.

I'll think about it and open a follow-up issue or PR.

**Comment by sbueringer** [comment]
@fabriziopandini sorry edited my comment :)

**Comment by fabriziopandini** [comment]
@sbueringer PTAL #7823

**Pull request merged**